sdk/data/azcosmos/cosmos_global_endpoint_manager.go

```diff
@@ -0,0 +1,163 @@
+package azcosmos
```

Comment:
File is missing the MIT license header.  You can copy it from [here](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/core.go#L4-L5) (note the newline between the header text and the package declaration).

Reply:
This doesn't seem to be fixed?

Reply:
There needs to be a space between the license header comment and the package name definition else the license comment shows up as a doc comment which we don't want.